go/api/base_client.go

```diff
@@ -626,6 +626,15 @@ func (client *baseClient) SPop(key string) (Result[string], error) {
 	return handleStringResponse(result)
 }
 
+func (client *baseClient) SMIsMember(key string, members []string) ([]Result[bool], error) {
+	result, err := client.executeCommand(C.SMIsMember, append([]string{key}, members...))
+	if err != nil {
+		return nil, err
```

Comment:
This returns Result bool type:
Is the return type correct?
If your return type is Nil bool result, please use CreateNilBoolResult() function.
There is a list of functions in response_types.go to create return types for Result struct type.
```suggestion
		return CreateNilBoolResult(), err
```

Reply:
This doesn't return Result bool type, but a slice (list) of it, and a slice in go can be nil.